src/helpers/getLanguagesValue.ts

```diff
@@ -0,0 +1,9 @@
+export const getLanguagesValue = (languages:{ [key: string]: string }) => {
```

Comment:
Siempre es mejor hacer referencia a los types que ya existen para no repetir el mismo código:

```suggestion
export const getLanguagesValue = (languages: CountriesData["languages"]) => {
```